pyproject.toml

```
@@ -67,7 +67,7 @@ mkdocs-git-authors-plugin = {version="*", optional=true}
 mkdocs-gitbook = {version="*", optional=true}
 
```

```suggestion
mkdocs-glightbox = {version="*", optional=true}

```
You also need to specify it as an optional dependency.